apps/judicial-system/backend/src/app/messages/notifications.ts

```diff
@@ -604,6 +604,13 @@ export const notifications = {
       description:
         'Texti í pósti til dómara og dómritara þegar að mál er kært til landsréttar',
     },
+    text: {
+      id: 'judicial.system.backend:notifications.case_appealed_to_court_of_appeals.text',
+      defaultMessage:
+        'Úrskurður hefur verið kærður í máli {courtCaseNumber}. Sjá nánar á rettarvorslugatt.island.is',
```

Comment:
Are we not sending links in SMS's?

Reply:
All our SMS's are sent with the link like that, it would be cool to look into possible ways to send a nice link straight to the case though!